src/main/java/com/adyen/model/checkout/PaymentSessionRequest.java

```diff
@@ -1383,195 +1462,102 @@ public boolean equals(java.lang.Object o) {
-                && Objects.equals(this.merchantRiskIndicator, paymentSessionRequest.merchantRiskIndicator)
-                && Objects.equals(this.metadata, paymentSessionRequest.metadata)
-                && Objects.equals(this.mpiData, paymentSessionRequest.mpiData)
-                && Objects.equals(this.nationality, paymentSessionRequest.nationality)
-                && Objects.equals(this.orderReference, paymentSessionRequest.orderReference)
-                && Objects.equals(this.origin, paymentSessionRequest.origin)
-                && Objects.equals(this.recurring, paymentSessionRequest.recurring)
-                && Objects.equals(this.recurringProcessingModel, paymentSessionRequest.recurringProcessingModel)
-                && Objects.equals(this.reference, paymentSessionRequest.reference)
-                && Objects.equals(this.returnUrl, paymentSessionRequest.returnUrl)
-                && Objects.equals(this.sdkVersion, paymentSessionRequest.sdkVersion)
-                && Objects.equals(this.selectedBrand, paymentSessionRequest.selectedBrand)
-                && Objects.equals(this.selectedRecurringDetailReference, paymentSessionRequest.selectedRecurringDetailReference)
-                && Objects.equals(this.sessionId, paymentSessionRequest.sessionId)
-                && Objects.equals(this.sessionValidity, paymentSessionRequest.sessionValidity)
-                && Objects.equals(this.shopperEmail, paymentSessionRequest.shopperEmail)
-                && Objects.equals(this.shopperIP, paymentSessionRequest.shopperIP)
-                && Objects.equals(this.shopperInteraction, paymentSessionRequest.shopperInteraction)
-                && Objects.equals(this.shopperLocale, paymentSessionRequest.shopperLocale)
-                && Objects.equals(this.shopperName, paymentSessionRequest.shopperName)
-                && Objects.equals(this.shopperReference, paymentSessionRequest.shopperReference)
-                && Objects.equals(this.shopperStatement, paymentSessionRequest.shopperStatement)
-                && Objects.equals(this.socialSecurityNumber, paymentSessionRequest.socialSecurityNumber)
-                && Objects.equals(this.splits, paymentSessionRequest.splits)
-                && Objects.equals(this.store, paymentSessionRequest.store)
-                && Objects.equals(this.storePaymentMethod, paymentSessionRequest.storePaymentMethod)
-                && Objects.equals(this.telephoneNumber, paymentSessionRequest.telephoneNumber)
-                && Objects.equals(this.threeDS2RequestData, paymentSessionRequest.threeDS2RequestData)
-                && Objects.equals(this.token, paymentSessionRequest.token)
-                && Objects.equals(this.totalsGroup, paymentSessionRequest.totalsGroup)
-                && Objects.equals(this.trustedShopper, paymentSessionRequest.trustedShopper)
-                && Objects.equals(this.uniqueTerminalId, paymentSessionRequest.uniqueTerminalId);
```

Comment:
uniqueTerminalId is still present in the file, should we add it back to the equals/hashcode methods or remove it from the file?

same for totalsGroup, accountInfo, additionalAmount, bankAccount, browserInfo, card, configId, deviceFingerprint, html, merchantRiskIndicator, mpiData, nationality, recurring, recurringProcessingModel, sdkVersion, selectedBrand, selectedRecurringDetailReference, sessionId, store, threeDS2RequestData, totalsGroup

Reply:
I added it back, it was a mistake on my part when generating it.